Corgibytes.Freshli.Cli/Functionality/Message/MessageBus.cs

```diff
@@ -1,27 +1,31 @@
 using System;
-using System.Reactive.Concurrency;
-using System.Reactive.Linq;
-using System.Reactive.Subjects;
+using Hangfire;
+using Hangfire.Server;
 
 namespace Corgibytes.Freshli.Cli.Functionality.Message;
 
 public class MessageBus
 {
-    private readonly IConnectableObservable<IMessage> _connectableObservable;
-    private readonly Subject<IMessage> _jobs = new();
+    private readonly IBackgroundProcessingServer _jobClient;
 
-    public MessageBus()
+    public MessageBus(IBackgroundProcessingServer jobClient)
     {
-        _connectableObservable = _jobs.ObserveOn(Scheduler.Default).Publish();
-        _connectableObservable.Connect();
+        _jobClient = jobClient;
         SubscribeHandlers();
     }
 
-    // ReSharper disable once UnusedMember.Global
-    public void Dispatch(IMessage job) => _jobs.OnNext(job);
+    public void Dispatch(IMessage job)
+    {
+        BackgroundJob.Enqueue(() => new TestMessageHandler().Handle(job));
```

Comment:
This is just for testing purposes to get Hangfire working, because the dispatch shouldn't be aware of the handler. It should throw the message into the queue and be done with it.